src/invidious.cr

```diff
@@ -116,11 +116,6 @@ end
 OUTPUT = CONFIG.output.upcase == "STDOUT" ? STDOUT : File.open(CONFIG.output, mode: "a")
 LOGGER = Invidious::LogHandler.new(OUTPUT, CONFIG.log_level)
 
-# Run migrations
-if Invidious::Database::Migrator.new(PG_DB).pending_migrations?
-  puts "There are pending migrations. Run `invidious --migrate` to apply the migrations."
-  exit 46
-end
```

Comment:
"This is so that we don't break deploys with this PR."

Well, when we'll make any DB changes (like #2469), the code won't correspond to the DB anymore, so invidious will still be broken until the migration is done.

Reply:
As an in-between, we could keep the check + message, but not exit (so it's easier to trace why invidious is broken).

Reply:
I made this change because it seems like the rollout of requiring migrations to have run needs more thought which would could give more attention in a followup PR. By removing that, we can merge in the migration code without worrying about breaking users